content/foundations/color.mdx

```diff
@@ -2,75 +2,188 @@
-
-### Global roles
-
-| Prefix | Role |
-| -------- | -------- |
-| `neutral-` | Use to highlight content without any added meaning. |
-| `accent-` | Use to draw attention to interactive elements. |
-| `success-` | Use to expresses the completion or positive outcome of a task. |
-| `attention-` | Use to warn of pending tasks or highlight active content.  |
-| `severe-` | Use when there are more than 3 levels of states, for example in heatmaps. |
-| `danger-` | Use to inform of error or another negative message. |
-
-### GitHub-specific roles
-
-| Prefix | Role |
-| -------- | -------- |
-| `done-` | Completion color for productivity and code review workflows. |
-| `sponsors-` | Color suite for all Sponsors-related content. |
-
-### Variants
-
-| Suffix | Role |
-| -------- | -------- |
-| `-fg` | Text and icons. |
-| `-emphasis` | High contrast for background and borders. |
-| `-emphasis-plus` | Highest contrast for background and borders. Use only with the `neutral-` prefix. |
-| `-muted` | Medium contrast alpha value, recommended for borders. |
-|`-subtle` | Low contrast solid color, recommended for backgrounds. |
-
-## How to use Primer colors?
+## How to use color for product UI
+Primer delivers colors in the form of [variables](https://primer.style/primitives/colors) or design tokens. Color variables are grouped based on their purpose:
+
+- **Functional:** To convey a meaning or a state. For example, from a functional perspective the color green is used to reinforce positive messaging. In a functional system, green variables are named with the suffix `.success`.
+- **Presentational:** To represent a color. For example, the color steps in the Primer scale are named by color and lightness, such as `scale.blue.5`. These variables don't currently support color modes. There is no available system to use colors in this form yet.
+
+As the system grows, it will provide more APIs that fit different color needs. Check the [Primer Primitives](https://github.com/primer/primitives/releases/) repository to follow along as we release new variables and systems.
+
+### Functional system
+The functional system is based on the meaning, or purpose, that colors have in the interface. This system is structured into two groups of variables:
+
+- **Foundations:** Foregrounds, backgrounds, and borders that make up most of a product interface. Each variable is generated based on visual weight. 
+- **Color roles:** Foregrounds, backgrounds, and borders that highlight affordance or the meaning of elements in the UI.
+
+<img width="960" alt="color-roles-list" src="https://user-images.githubusercontent.com/6951037/141302835-05ea2e26-fc89-49ed-8dc5-2af6a6ca9df8.png" />
+
+You should always use Primer's functional color system in your designs to make sure that:
+
+ - It supports color modes out of the box
+ - It gets all future themes updates "for free"
+ - You have an accessible color answer across themes
```

Comment:
I think this line means that your color choices will remain accessible through the color themes if you're using correct primer variables, but the wording of "accessible color answer" feels a bit awkward. How about something like this? 
```suggestion
 - You have an accessible color combination across themes
```